config/dynamic.c

```diff
@@ -50,6 +50,8 @@ static const uint16_t MSG_MAX_LEN = (uint16_t)~0U;
 
 static struct dynamic_config config;
 
+extern const struct luaL_reg dylib_lua_c_funcs [];
+
```

Comment:
This must be placed in a header file. Notice that since `dylib_lua_c_funcs` is a pointer, the header file does not have to define `struct luaL_reg`.

Reply:
Could you try `extern const struct luaL_reg *dylib_lua_c_funcs;` in the header file instead of declaring `dylib_lua_c_funcs` as an array there?

Reply:
If so, the error would be ``error: conflicting types for ‘dylib_lua_c_funcs’``